app/src/main/java/com/duckduckgo/app/browser/webview/MaliciousSiteBlockerWebViewIntegration.kt

```diff
@@ -174,4 +204,17 @@ class RealMaliciousSiteBlockerWebViewIntegration @Inject constructor(
     override fun onPageLoadStarted() {
         processedUrls.clear()
     }
+
+    override fun onSiteExempted(url: Uri) {
+        val convertedUrl = URLDecoder.decode(url.toString(), "UTF-8").lowercase()
+        exemptedUrlsHolder.exemptedMaliciousUrls.add(convertedUrl)
+        Timber.tag("MaliciousSiteDetector").d(
+            "Added $url to exemptedUrls, contents: ${exemptedUrlsHolder.exemptedMaliciousUrls}",
+        )
+    }
+
+    override fun onFireButtonPressed() {
```

Comment:
Since URLs are stored in memory and Fire button forces an app restart, we don't really need this